docker-compose.yml

```
@@ -575,7 +575,7 @@ services:
   jaeger:
     image: jaegertracing/all-in-one
     container_name: jaeger
-    command: ["--memory.max-traces", "10000", "--query.base-path", "/jaeger/ui"]
+    command: ["--prometheus.server-url","http://${PROMETHEUS_ADDR}","--memory.max-traces", "10000", "--query.base-path", "/jaeger/ui"]
```

With this expanding to several parameters, let's make it a multiline array instead:

```suggestion
    command: 
      - "--memory.max-traces"
      - "10000"
      - "--query.base-path"
      - "/jaeger/ui"
      - "--prometheus.server-url"
      - "http://${PROMETHEUS_ADDR}"
```